src/engine/renderer/tr_local.h

```diff
@@ -1108,6 +1113,10 @@ enum class dynamicLightRenderer_t { LEGACY, TILED };
 		bool shaderHasNoLight;
 
 		stageRenderer_t colorRenderer;
+		stageSurfaceDataUpdater_t surfaceDataUpdater;
```

Comment:
The material renderer / non-material renderer/ legacy dynamic light rendering bits are all jumbled together here. Could use some comments to clarify which is which

Reply:
Yes, I wanted to know how to name the non-material renderer before adding some comments, now done. 😉️